tests/ContextTest.py

```diff
@@ -95,5 +97,103 @@ def test_add_import(self):
         ctx.add_import(ctx2_1)
         ctx3 = Context(ident='http://example.com/context_3')
         ctx3.add_import(ctx)
-        final_ctx = Context(ident='http://example.com/context_1', imported=(ctx3,))
+        final_ctx = Context(
+            ident='http://example.com/context_1', imported=(ctx3,))
         self.assertEqual(len(final_ctx.save_imports()), 4)
+
+    def test_init_len(self):
+        ctx = Context(ident='http://example.com/context_1')
+        self.assertEqual(len(ctx), 0)
+
+    def test_len(self):
+        ident_uri = 'http://example.com/context_1'
+        ctx = Context(ident=ident_uri)
+        ctx.add_import(Context(ident=ident_uri))
+        for i in range(0, 5):
+            ctx.add_statement(create_mock_statement(ident_uri, i))
+        self.assertEqual(len(ctx), 5)
+
+    def test_add_remove_statement(self):
+        ident_uri = 'http://example.com/context_1'
+        ctx = Context(ident=ident_uri)
+        stmt1 = create_mock_statement(ident_uri, 1)
```

Comment:
Why not use a list comprehension for this?

Reply:
Because I want to remove statement in the middle of the list. But after all, it's a little bit useless. I'm changing it.